README.md

```diff
@@ -142,6 +163,38 @@ hosts=host_list_provider(hosts=["some.host", "172.100.100.20"], ssh_config=ssh)
 ...
 ```
 
+### SSH-Agent for ssh and scp connections
```

Comment:
I don't think this makes sense here, let's move that block here - https://github.com/vmware-tanzu/crash-diagnostics/tree/main/docs#crashd_config at the end of that section (crashd_config).